api_v3/services/PartnerService.php

```diff
@@ -141,6 +141,12 @@ public function registerAction( KalturaPartner $partner , $cmsPassword = "" , $t
 			$exceptionMessage = (str_replace('KalturaPartner::', '', $ex->getMessage()));
 			throw new KalturaAPIException( APIErrors::PARTNER_REGISTRATION_ERROR, $exceptionMessage);
 		}
+		catch (KalturaAPIException $ex)
+		{
+			KalturaLog::CRIT($ex);
+			$exceptionMessage = (str_replace('KalturaPartner::', '', $ex->getMessage()));
```

Comment:
i think that the error can come from multiple classes so lets remove this handling from here and from the core exception block and just throw $ex->getMessage()